frontend/app/MainTabs.tsx

```diff
@@ -11,24 +11,24 @@ import {createBottomTabNavigator} from '@react-navigation/bottom-tabs';
 import Icon from 'react-native-vector-icons/Ionicons';
 import HomeScreen from './home/HomeScreen';
 import ExploreScreen from './explore/ExploreScreen';
-import ScanScreen from './scan/ScanScreen';
-import FriendsScreen from './friends/FriendsScreen';
+import ScanTab from './scan/ScanTab';
+import FriendsTab from './friends/FriendsTab';
 import ProfileScreen from './profile/ProfileScreen';
 
 export type MainTabsParamList = {
   Home: undefined;
-  Scan: undefined;
+  'Scan Tab': undefined;
```

Comment:
Why just some of them are saves as strings? I get the error when trying to navigate to Saved Items that no such screen is available ("Saved Items" was not handled by any Navigator)

Reply:
They're basically all strings. You just need to add the quotes when there are spaces.